webapp/channels/src/components/widgets/advanced_textbox/advanced_textbox.tsx

```diff
@@ -51,6 +52,7 @@ const AdvancedTextbox = ({
     onValidate,
     showCharacterCount = false,
     readOnly = false,
+    classname,
```

Comment:
I am failing to see where this new prop is used. Maybe it was intended to have a purpose but it no longer has?

Reply:
Removed, it was no longer being used.